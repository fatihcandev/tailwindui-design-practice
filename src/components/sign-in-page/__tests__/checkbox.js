import React from 'react'
import renderer from 'react-test-renderer'

import Checkbox from '../checkbox'

describe('Checkbox', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Checkbox />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
