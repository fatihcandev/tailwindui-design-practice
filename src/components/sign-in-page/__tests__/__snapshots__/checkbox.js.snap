// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox renders correctly 1`] = `
<div
  className="checkboxWrapper"
>
  <button
    className="checkbox"
  >
    <svg
      fill="none"
      stroke="currentColor"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth="2"
      viewBox="0 0 24 24"
    >
      <path
        d="M5 13l4 4L19 7"
      />
    </svg>
  </button>
  <p />
</div>
`;
